chenjefferson/photo-picker
# Setup Eslint

## Rationale

We want ESLint and tests to run on each commit and before pushing.

## Implementation

This PR adds a husky pre-commit hook that runs linting and tests, and configures `eslint` to work with React Native.

## Testing Instructions

1. Run `npm run test` -- this should give you check marks.
2. Run `npm run lint`. Add in some lint errors (ex. spaces at the end of a line, ex. in `index.tsx`) and re-run. Check that errors appear.
3. Run `npm run prepare`. This will run your hooks.
4. Make your first commit! Update the `README.md` and add your name under a section titled `Contributors`. To make a title in markdown, use `##`, ex. `## This Is the Section Header`.
    - Check that the linting and tests run.
